src/hooks/usePhaseData.ts

```diff
@@ -121,18 +119,18 @@ export const usePhaseData = (phaseEnum?: PhaseEnum | null): GetPhaseData => {
 
   // activityTypes
   // {INITIATION_SPECIAL_GIFT_APPEAL: {
-  //   name: 'Special Gift Appeal',
+  //   translatedShortName: 'Special Gift Appeal',
+  //   translatedFullName: 'Initiation - Special Gift Appeal',
   //   phase: 'Initiation',
   //   phaseId: 'INITIATION',
   //   subject: 'Special Gift Appeal',
-  //   title: 'Initiation - Special Gift Appeal',
   // },
   // FOLLOW_UP_TEXT_MESSAGE: {
-  //   name: 'Text Message',
+  //   translatedShortName: 'Text Message',
+  //   translatedFullName: 'Follow Up - Text Message',
   //   phase: 'Follow-Up',
   //   phaseId: 'FOLLOW_UP',
   //   subject: 'Text Message To Follow Up',
-  //   title: 'Follow Up - Text Message',
   // },}
 
   const activityTypes: Map<ActivityTypeEnum, ActivityData> = useMemo(() => {
```

Comment:
Can you move the `Map` type to where you do `new Map()`? It's letting you put `translatedFullName: string | undefined` into the map because the map is currently of `<any, any>`.

```suggestion
  const activityTypes = useMemo(() => {
    const activitiesMap = new Map<ActivityTypeEnum, ActivityData>();
```